src/BlazorBoilerplate.Server/Startup.cs

```diff
@@ -342,12 +342,6 @@ void DbContextOptionsBuilder(DbContextOptionsBuilder builder)
                 };
             });
 
-            services.AddResponseCompression(opts =>
```

Comment:
Is there a reason to remove? I think I tried removing it from a previous PR and I was getting errors. I will research.